### PR TITLE
Define validators in the `cli` module.

### PR DESCRIPTION
**Pull request by IlliaShepilov**
Summary of changes:
- Defined `cli` module.
- Created validators for the `cli` module needs.
- Created utility classes for processing the validators.

### CODE REVIEW

**Comment by IlliaShepilov** [comment]
@armiol PTAL.

File: cli/src/main/java/org/spine3/examples/todolist/validator/ApproveValidator.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * Serves as a validator for the user approve answer.
+ *
+ * <p>Validation will be passed when:
+ *    <li>input is `y`;
+ *    <li>input is `n`.
+ * <p>In other cases validation will be failed.
+ *
+ * @author Illia Shepilov
+ */
+public class ApproveValidator implements Validator {
```
Comment (armiol) [inline cli/src/main/java/org/spine3/examples/todolist/validator/ApproveValidator.java]: With this name it validates `approves`. It does another job, according to the description.

Please rename.

File: cli/src/main/java/org/spine3/examples/todolist/validator/ApproveValidator.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    private static final String INCORRECT_INPUT = "Incorrect input. Valid values: 'y' or 'n'";
```
Comment (armiol) [inline cli/src/main/java/org/spine3/examples/todolist/validator/ApproveValidator.java]: Should probably be `Invalid`, not `Incorrect`.

File: cli/src/main/java/org/spine3/examples/todolist/validator/CommonValidator.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+package org.spine3.examples.todolist.validator;
+
+import javax.annotation.Nullable;
+
+import static org.spine3.examples.todolist.validator.ValidatorHelper.isEmpty;
+import static org.spine3.examples.todolist.validator.ValidatorHelper.isNull;
+
+/**
+ * Serves as a validator when is needed validate common cases.
```
Comment (armiol) [inline cli/src/main/java/org/spine3/examples/todolist/validator/CommonValidator.java]: `to validate the`

File: cli/src/main/java/org/spine3/examples/todolist/validator/CommonValidator.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ * <p>Validation will be failed when input neither empty or {@code null}.
+ *
+ * <p>In other cases validation will be passed.
```
Comment (armiol) [inline cli/src/main/java/org/spine3/examples/todolist/validator/CommonValidator.java]: `skipped` or `will pass successfully`?

File: cli/src/main/java/org/spine3/examples/todolist/validator/CommonValidator.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+import javax.annotation.Nullable;
+
+import static org.spine3.examples.todolist.validator.ValidatorHelper.isEmpty;
+import static org.spine3.examples.todolist.validator.ValidatorHelper.isNull;
+
+/**
+ * Serves as a validator when is needed validate common cases.
+ *
+ * <p>Validation will be failed when input neither empty or {@code null}.
```
Comment (armiol) [inline cli/src/main/java/org/spine3/examples/todolist/validator/CommonValidator.java]: Something is wrong with this sentence. Let's discuss.

File: cli/src/main/java/org/spine3/examples/todolist/validator/CommonValidator.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ * <p>In other cases validation will be passed.
+ *
+ * @author Illia Shepilov
+ */
+public class CommonValidator implements Validator {
```
Comment (armiol) [inline cli/src/main/java/org/spine3/examples/todolist/validator/CommonValidator.java]: Let's rename. If its purpose is to infiltrate only those values which aren't empty or `null`, it should be named accordingly.

**Pull request merged**